src/components/Avatar.vue

```diff
@@ -0,0 +1,48 @@
+<!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
i guess this should be put right above the `img` tag or disabled in file (now disable-next-line points to `<template>`).